packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -78,9 +152,13 @@ import 'theme.dart';
 /// `Navigator.of(rootNavigator: true)` from inside the [BuildContext] of a
 /// [CupertinoTabView].
 ///
+/// To manage the current tab from an ancestor widget, use a [CupertinoTabController]
+/// and pass it to the descendent [CupertinoPageScaffold].
+///
 /// See also:
 ///
 ///  * [CupertinoTabBar], the bottom tab bar inserted in the scaffold.
+///  * [CupertinoTabController], the exposed current page state of a [CupertinoPageScaffold].
```

Comment:
Is this right? CupertinoTabController shouldn't be related to _CupertinoPageScaffoldState